Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM resin/rpi-raspbian:wheezy-2015-01-15
-
+FROM resin/rpi-raspbian:jessie
 # Install Python.
-RUN apt-get update && apt-get install -y python
+RUN apt-get update 
+	&& apt-get install -y python 
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
@nghiant2710 how much size does this reduce? it may confuse beginners? one solution may be to put a comment and explain what is going on here